apps/liteprotocoltester/diagnose_connections.nim

```diff
@@ -4,93 +4,74 @@ else:
   {.push raises: [].}
 
 import
-  std/[options, strutils, os, sequtils, net, strformat],
+  std/[options, net, strformat],
   chronicles,
   chronos,
   metrics,
   libbacktrace,
-  system/ansi_c,
   libp2p/crypto/crypto,
   confutils,
   libp2p/wire
 
 import
-  ../../waku/common/logging,
-  ../../waku/factory/waku,
-  ../../waku/factory/external_config,
-  ../../waku/node/health_monitor,
-  ../../waku/node/waku_metrics,
-  ../../waku/waku_api/rest/builder as rest_server_builder,
-  ../../waku/node/peer_manager,
-  ../../waku/waku_lightpush/common,
-  ../../waku/waku_relay,
-  ../../waku/waku_filter_v2,
-  ../../waku/waku_api/rest/client,
-  ../../waku/waku_api/rest/admin/client,
-  ./tester_config,
-  ./lightpush_publisher,
-  ./filter_subscriber
-
+  waku/[
+    factory/external_config,
+    node/peer_manager,
+    waku_lightpush/common,
+    waku_relay,
+    waku_filter_v2,
+    waku_peer_exchange/protocol,
+    waku_core/multiaddrstr,
+    waku_enr/capabilities,
+  ]
 logScope:
   topics = "diagnose connections"
 
-proc logSelfPeersLoop(pm: PeerManager, interval: Duration) {.async.} =
-  trace "Starting logSelfPeersLoop diagnosis loop"
-  while true:
-    let selfLighpushPeers = pm.wakuPeerStore.getPeersByProtocol(WakuLightPushCodec)
-    let selfRelayPeers = pm.wakuPeerStore.getPeersByProtocol(WakuRelayCodec)
-    let selfFilterPeers = pm.wakuPeerStore.getPeersByProtocol(WakuFilterSubscribeCodec)
+proc `$`*(cap: Capabilities): string =
+  case cap
+  of Capabilities.Relay:
+    return "Relay"
+  of Capabilities.Store:
+    return "Store"
+  of Capabilities.Filter:
+    return "Filter"
+  of Capabilities.Lightpush:
+    return "Lightpush"
+  of Capabilities.Sync:
+    return "Sync"
+
+proc allPeers(pm: PeerManager): string =
```

Comment:
I wonder if we should move these "PeerManager" procs to the `peer_manager` module, or nearby.